Doc/library/ssl.rst

```diff
@@ -1729,6 +1779,26 @@ to speed up repeated connections from the same clients.
 
      This features requires OpenSSL 0.9.8f or newer.
 
+.. attribute:: SSLContext.highest_version
+
+   An :class:`TLSversion` enum member representing the highest supported
+   TLS version. The value defaults to :attr:`TLSVersion.MAXIMUM_SUPPORTED`.
+   The flag and :attr:`SSLContext.options` both affect the supported SSL
+   and TLS versions of the context.
+
+   .. note::
+
+     This features requires OpenSSL 1.1.0g or newer.
```

Comment:
I'm not sure what this means :-)

Reply:
The feature is not available unless the ssl module is compiled with OpenSSL 1.1.0g or a later version. OpenSSL 1.1.0f and older versions do not have getters.

Reply:
I think something like "This attribute does not exist unless ..." would be clearer